src/auth/auth.service.ts

```diff
@@ -0,0 +1,12 @@
+import { Injectable } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+
+@Injectable()
+export class AuthService {
+  constructor(private configService: ConfigService) {}
```

Comment:
we can make the `configService` `readonly` as well

Reply:
this is updated now